src/js/lib/jquery.kazoosdk.js

```diff
@@ -126,6 +126,23 @@
 			contactList: {
 				'get': { verb: 'GET', url: 'accounts/{accountId}/contact_list' }
 			},
+			crm: {
+				'getIntegration': { verb: 'GET', url: 'accounts/{accountId}/crm_integration' },
+				'createIntegration': { verb: 'PUT', url: 'accounts/{accountId}/crm_integration' },
+				'getTrigger': { verb: 'GET', url: 'accounts/{accountId}/crm_integration/triggers/{triggerId}' },
+				'listTrigger': { verb: 'GET', url: 'accounts/{accountId}/crm_integration/triggers'},
```

Comment:
Let's keep the resource plural on listings to keep things logical and make it easier to remember:
```suggestion
				'listTriggers': { verb: 'GET', url: 'accounts/{accountId}/crm_integration/triggers'},
```